src/main/java/uk/gov/ons/ctp/response/action/representation/exception/InvalidSituationException.java

```diff
@@ -0,0 +1,19 @@
+package uk.gov.ons.ctp.response.action.representation.exception;
+
+import uk.gov.ons.ctp.common.error.CTPException;
+import uk.gov.ons.ctp.response.action.representation.Situation;
+
+public class InvalidSituationException extends CTPException {
```

Comment:
Should this be a checked exception, is it recoverable?

Reply:
We were trying to follow what seemed to be convention in the code bases. As far as we can see, any occurrence of this exception would be a logic error (i.e. code mistake). Do you think we should change this to a `RuntimeException`?

Reply:
I reckon it's worth it being a RuntimeException. Saves the next person using it being confused